app/settings.coverage.json

```diff
@@ -0,0 +1,13 @@
+
+{
+  "coverage": {
+    "coverage_app_folder": "/Users/Leo/WebstormProjects/winsleague/app/",
```

Comment:
@serut how do we make this dynamic? Can we use `${PWD/#$HOME/~}/`? This works in my zsh shell but not sure how supported it is elsewhere.